pkg/cmd/server/api/v1/types.go

```diff
@@ -702,93 +702,100 @@ type LDAPSyncConfig struct {
 	api.TypeMeta `json:",inline"`
 	// Host is the scheme, host and port of the LDAP server to connect to:
 	// scheme://host:port
-	Host string `json:"host" description:"scheme://host:port for the LDAP server"`
+	URL string `json:"url" description:"scheme://host:port for the LDAP server"`
 	// BindDN is an optional DN to bind to the LDAP server with
-	BindDN string `json:"bindDN,omitempty" description:"the optional DN to bind with"`
+	BindDN string `json:"bindDN" description:"the optional DN to bind with"`
 	// BindPassword is an optional password to bind with during the search phase.
-	BindPassword string `json:"bindPassword,omitempty" description:"the optional password to bind with"`
+	BindPassword string `json:"bindPassword" description:"the optional password to bind with"`
 	// Insecure, if true, indicates the connection should not use TLS.
 	// Cannot be set to true with a URL scheme of "ldaps://"
 	// If false, "ldaps://" URLs connect using TLS, and "ldap://" URLs are upgraded to a TLS connection using StartTLS as specified in https://tools.ietf.org/html/rfc2830
 	Insecure bool `json:"insecure" description:"specifies that the connection with the server should not use TLS"`
 	// CA is the optional trusted certificate authority bundle to use when making requests to the server
 	// If empty, the default system roots are used
-	CA string `json:"CA,omitempty" description:"an optional trusted CA to use when making requests to the server"`
+	CA string `json:"ca" description:"an optional trusted CA to use when making requests to the server"`
 
 	// LDAPGroupUIDToOpenShiftGroupNameMapping is an optional direct mapping of LDAP group UIDs to
 	// OpenShift Group names
-	LDAPGroupUIDToOpenShiftGroupNameMapping map[string]string
+	LDAPGroupUIDToOpenShiftGroupNameMapping map[string]string `json:"ldapGroupUIDToOpenShiftGroupNameMapping" description:"optional direct mapping of LDAP group UIDs to OpenShift Group names"`
```

Comment:
This began as a subtle dig against @deads2k asking me to be specific every time I wasn't... I'm excited to see what this ends as